fhir-server/src/main/java/com/ibm/fhir/server/rest/FHIRRestInteractionVisitorBase.java

```diff
@@ -78,40 +78,42 @@ protected FHIRRestInteractionVisitorBase(FHIRResourceHelpers helpers,
      * @param entryIndex
      * @param e
      * @param requestDescription
-     * @param initialTime
+     * @param accumulatedTime
      */
-    protected void setEntryComplete(int entryIndex, Entry e, String requestDescription, long initialTime) {
+    protected void setEntryComplete(int entryIndex, Entry e, String requestDescription, long accumulatedTime) {
         responseBundleEntries[entryIndex] = e;
-        logBundledRequestCompletedMsg(requestDescription, initialTime, e.getResponse().getStatus().getValue());
+        logBundledRequestCompletedMsg(requestDescription, accumulatedTime, e.getResponse().getStatus().getValue());
     }
 
     protected Entry getResponseEntry(int entryIndex) {
         return responseBundleEntries[entryIndex];
     }
 
-    private void logBundledRequestCompletedMsg(String requestDescription, long initialTime, String httpStatus) {
-        StringBuffer msg = new StringBuffer();
-        double elapsedSecs = (System.currentTimeMillis() - initialTime) / 1000.0;
-
-        msg.append("Completed bundle request took:[");
-        msg.append(elapsedSecs);
-        msg.append(" secs]: ");
-        msg.append(requestDescription);
-        msg.append(" status:[" + httpStatus + "]");
-        log.info(msg.toString());
+    private void logBundledRequestCompletedMsg(String requestDescription, long accumulatedTime, String httpStatus) {
+        double elapsedSecs = accumulatedTime / 1000000000.0;
+
+        final String msg = String.format("Completed bundle request took:[%7.3f secs]: %s status:[%s]", 
+            elapsedSecs, requestDescription, httpStatus);
+        log.info(msg);
+//        StringBuffer msg = new StringBuffer();
+//        msg.append("Completed bundle request took:[");
+//        msg.append(elapsedSecs);
+//        msg.append(" secs]: ");
+//        msg.append(requestDescription);
+//        msg.append(" status:[" + httpStatus + "]");
```

Comment:
I assume its just a preference thing.  If we go with String.format, please remove the commented lines

Reply:
I wanted to format the double, and the String.format is just a lot more compact. I had intended to remove the commented lines after checking it didn't break anything. Will do so now.